test/service/test_pypi.py

```diff
@@ -7,6 +7,7 @@
 from packaging.version import Version
 
 import pip_audit.service as service
+from pip_audit.service.pypi import _get_cache_dir
```

Comment:
This is testing a private helper that isn't used in other modules. We tend not to do this, but I figured I should add some tests since I've already messed this up once.

If this isn't appropriate, feel free to revert the latest commit.